Bottom: 5ed6ac2ba2a2b20891f0ee10a1da06e4c1364ee4
Top:    af14679008ba1456d3b60184c849e3c8a2b67daa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-25 14:09:21 +0100

Change scroll-wheel bindings to match more GTK applications










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index e9469e9..7b0769d 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
